### ISSUE TITLE
Plist: generates incorrect Bool values

### ISSUE THREAD

**Issue by kacper1703** [SwiftGen/SwiftGen#775]
### Environment
- Xcode 12.0 GM
- SwiftGen 6.2.1 (via CocoaPods)

### Repro steps
- Create a plist `Example.plist`
- In the root dictionary add a Bool key `IsTestEnabled` with value `false` (which Xcode corrects to **0** and it cannot be bypassed)
- In swiftgen.yml:
```yaml
plist:
  - inputs: [PATH]/Example.plist
    outputs:
      - templateName: inline-swift5
        output: ExamplePlistKeys.swift
        params:
          enumName: "PlistKeys"
```

### Result
Compiler can't accept `0` as a Boolean value.
```swift
internal enum PlistKeys {
  internal enum Example {
    internal static let isTestEnabled: Bool = 0
  }
```

### Expected result
Boolean values are converted from `0` and `1` to `false` and `true`, respectively.

**Comment by lucasmpaim** [comment]
+1

**Comment by djbe** [comment]
I'm not 100% sure where the issue is here? From what I can see, we **have** tests for boolean values in `Plist` files, and they generate the correct output. See here for example:
- input plist: https://github.com/SwiftGen/SwiftGen/blob/stable/Tests/Fixtures/Resources/Plist/good/Info.plist#L39-L42
- output swift: https://github.com/SwiftGen/SwiftGen/blob/stable/Tests/Fixtures/Generated/Plist/inline-swift5/all.swift#L25-L26

**Comment by djbe** [comment]
Could you provide an example plist file? So we can test it ourselves?

**Comment by djbe** [comment]
Ah, seeing @AliSoftware's [latest comment](https://github.com/SwiftGen/SwiftGen/pull/779#issuecomment-699653383) explains it a bit better, the issue exists when **not** testing, where we do not have an YAML serialisation step in between parsing and output generation.

**Comment by kacper1703** [comment]
@djbe would you still like an example project?

**Comment by djbe** [comment]
Implemented in SwiftGen 6.4.0, which has been released 🎉
